monai/transforms/transforms.py

```diff
@@ -26,6 +28,84 @@
 export = monai.utils.export("monai.transforms")
 
 
+@export
+class LoadNifti:
+    """
+    Load Nifti format file from provided path.
+    """
+
+    def __init__(self, as_closest_canonical=False, image_only=False, dtype=None):
+        """
+        Args:
+            as_closest_canonical (bool): if True, load the image as closest to canonical axis format.
+            image_only (bool): if True return only the image volume, other return image volume and header dict.
+            dtype (np.dtype, optional): if not None convert the loaded image to this data type.
+        """
+        self.as_closest_canonical = as_closest_canonical
+        self.image_only = image_only
+        self.dtype = dtype
+
+    def __call__(self, filename):
+        """
+        Args:
+            filename (str or file): path to file or file-like object.
+
+        Returns:
+            The loaded image volume if `image_only` is True, or a tuple containing the volume and the Nifti
+            header in dict format otherwise.
+
+        Note:
+            header['original_affine'] stores the original affine loaded from `filename_or_obj`.
+            header['affine'] stores the affine after the optional `as_closest_canonical` transform.
+        """
+        img = nib.load(filename)
+
+        header = dict(img.header)
+        header['filename_or_obj'] = filename
+        header['original_affine'] = img.affine
+        header['affine'] = img.affine
+        header['as_closest_canonical'] = self.as_closest_canonical
+
+        if self.as_closest_canonical:
+            img = nib.as_closest_canonical(img)
+            header['affine'] = img.affine
+
+        if self.dtype is not None:
+            img = img.get_fdata(dtype=self.dtype)
+        else:
+            img = np.asanyarray(img.dataobj)
+
+        if self.image_only:
+            return img
+        compatible_meta = dict()
+        for meta_key in header:
+            meta_datum = header[meta_key]
+            if type(meta_datum).__name__ == 'ndarray' \
+                    and np_str_obj_array_pattern.search(meta_datum.dtype.str) is not None:
+                continue
+            compatible_meta[meta_key] = meta_datum
+        return img, compatible_meta
+
+
+@export
+class AsChannelFirst:
+    """
+    Change the channel dimension of the image to the first dimension.
+    Args:
+        channel_dim (int): which dimension of input image is the channel, default is the last dimension.
+    """
+
+    def __init__(self, channel_dim=-1):
+        self.channel_dim = channel_dim
+
+    def __call__(self, img):
```

Comment:
this could be implemented with `np.moveaxis`

Reply:
Thanks for your suggestion, replaced with np.moveaxis() API.